lib/primitives/object-element.js

```diff
@@ -45,6 +45,14 @@ module.exports = function(BaseElement, ArrayElement, MemberElement) {
       );
     },
 
+    delete: function (name) {
```

Comment:
It's the name of the key. This is the function signature being used everywhere. I didn't want to change it.